activerecord/test/cases/counter_cache_test.rb

```diff
@@ -239,6 +257,16 @@ class ::SpecialReply < ::Reply
     end
   end
 
+  test "run after_commit callback with touch: true" do
+    assert_equal ::Task.count, 0
+
+    topic = ::TopicWithAfterCommitCallback.create!(title: "Topic with after commit callback")
+    assert_equal ::Task.count, 1
+
+    ::ReplyWithTouch.create!(topic: topic)
+    assert_equal ::Task.count, 2
```

Comment:
This fails as `TopicWithAfterCommitCallback` `after_commit :increase_task_count` is not called